server/app/api/controllers/peer_review.js

```diff
@@ -95,11 +94,12 @@ module.exports = {
     Peer_Review_Model.findOneAndUpdate(
       { _id: req.params.id },
       {
-        status: "Inactive"
+        status: "Inactive",
+        updated_date : new Date(),
+        last_updated_by : req.user.userName,
       },
-      function (err, userInfo) {
+      function (err, info) {
```

Comment:
info is not being used

Reply:
Info was assumed as part of syntax as call back function. Removed as it's not being used and api is working fine without info parameter.